source/back.tex

```diff
@@ -29,6 +29,11 @@ \chapter{Bibliography}
   Edited by Mark Davis. Revision 33; issued for Unicode 13.0.0.
   2020-02-13 [viewed 2021-06-08].
   Available from: \url{https://www.unicode.org/reports/tr31/tr31-33.html}
+\item
+  The Unicode Standard Version 14.0,
+  \doccite{Core Specification}, Chapter 3.9.
```

Comment:
```suggestion
  \doccite{Core Specification}.
```
I think you call out sections of a document only at the point of reference, and here only the part of a series of documents.

Reply:
Fixed? I've made this change assuming you're right. Note that the literature references do things differently tho, e.g.
`  \doccite{The \Cpp{} Programming Language, second edition}, Chapter R.
`
Are the references to the "Chapter 3.9" in the wording correct as is?

Reply:
It's OK. The entry and the references added by the PR in its current form match https://www.iso.org/sites/directives/current/part2/index.xhtml#_idTextAnchor136 examples 1 and 2.

The one you quoted also complies with https://www.iso.org/sites/directives/current/part2/index.xhtml#_idTextAnchor296. It's not mentioned outside the bibliography, so it makes sense that it specifies a relevant section.